spam/03_spam_data_slicing_tutorial.py

```diff
@@ -326,7 +326,7 @@ def textblob_polarity(x):
 
 # For demonstration purposes, we set n_epochs=2
 trainer = Trainer(lr=1e-4, n_epochs=2)
-trainer.fit(slice_model, [train_dl, valid_dl])
+# trainer.fit(slice_model, [train_dl, valid_dl])
```

Comment:
Should this be commented out?

Reply:
nope, good catch! was testing more quickly